app/views/handlers/view_preview_questions_pdf.py

```diff
@@ -0,0 +1,59 @@
+import io
+import re
+from datetime import datetime, timezone
+
+import pdfkit
+from flask import current_app
+
+from app.data_models import QuestionnaireStore
+from app.questionnaire import QuestionnaireSchema
+from app.views.handlers.view_preview_questions import ViewPreviewQuestions
+
+
+class ViewPreviewQuestionsPDF(ViewPreviewQuestions):
```

Comment:
As discussed, we should avoid this duplication but making the PDF generation a generic class that view submitted response and preview page can both share.

Reply:
Added all this, thanks.